.golangci.yml

```diff
@@ -0,0 +1,26 @@
+linters:
+  auto-fix: false
```

Comment:
Please, auto-fix is very helpful. So, we should not add this. 
Currently, we have the make lint-fix which allow us to use this feature.